server/pyproject.toml

```diff
@@ -1,9 +1,34 @@
+[tool.poetry]
+name = "testflinger"
+description = "Testflinger Server"
+readme = "README.rst"
+version = "1.1.0"                  # specify your version here
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.10"                                        # specify your Python version requirement here
+apiflask = "^2.1.1"
+flask = "^3.0.3"
+flask-pymongo = "^2.3.0"
+gunicorn = "^22.0.0"
+gevent = "^24.2.1"
+prometheus-client = "^0.20.0"
+pyyaml = "^6.0.1"
+sentry-sdk = { extras = ["flask"], version = "^2.0.1" }
+
+[tool.poetry.dev-dependencies]
+pytest = "^8.1.2"
```

Comment:
should we add `pytest` dev dependency to agent too?

Reply:
It shows that I already added them, am I missing something that's different on the similar section of pyproject.toml for the agent?

Reply:
you're right, Idk how I missed it